microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -428,7 +436,7 @@ The PWAmp widget displays the current song and buttons to play the previous or n
 <!-- ====================================================================== -->
 <!-- widgets are too early in the making for MDN. But there is enough complexity with the APIs that a reference doc is needed.
 For now we are documenting all functions and objects in this section. When widgets become standard, move this to MDN and remove the section. -->
-## Service Worker API reference
+## Service worker API reference
 
```

Comment:
missing end-row | char on some rows, yet rendered ok.  fixed.

https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/widgets#service-worker-api-reference

Reply:
fresh Files Changed tab confirms this was done